web-app/js/portal/cart/DownloadPanelItemTemplate.js

```diff
@@ -137,45 +138,42 @@ Portal.cart.DownloadPanelItemTemplate = Ext.extend(Ext.XTemplate, {
     },
 
     _createRemoveButtonAfterPageLoad: function(collection) {
-        this._createRemoveButton.defer(1, this, [collection]);
+        this._createRemoveLink.defer(1, this, [collection]);
         return '';
     },
 
-    _createRemoveButton: function(collection) {
-        var elementId = this._getButtonId(collection, 'removeButtonId');
+    _createRemoveLink: function(collection) {
+        var elementId = this._getlinkId(collection, 'removeButtonId');
         if (collection.menuItems.length > 0 && Ext.get(elementId)) {
 
             // remove old button
             Ext.fly(elementId).update("");
 
-            new Ext.Button({
+            this.removeLink = new Ext.ux.Hyperlink({
                 text: OpenLayers.i18n("removeButton"),
-                tooltip: OpenLayers.i18n("removeButtonTooltip"),
-                width: 65,
-                scope: this,
-                renderTo: elementId,
-                listeners: {
-                    click: {
-                        fn: this._removeButtonOnClick,
-                        scope: this
-                    }
-                }
+                title: OpenLayers.i18n("removeButtonTooltip"),
+                cls: 'x-tool-awesome fa fa-fw fa-close',
+                renderTo: elementId
             });
+            this.removeLink.on('click', function() {
+                this._removeOnClick(elementId);
+            }, this);
+
+
         }
     },
 
-    getIdFromButtonContainerId: function(button, i18Name) {
-        var collectionId = button.container.id;
+    getIdFromButtonContainerId: function(containerId, i18Name) {
         var prefix = OpenLayers.i18n(i18Name, {id: ""});
-        return collectionId.replace(prefix, '');
+        return containerId.replace(prefix, '');
     },
 
-    _getButtonId: function(collection, i18Name) {
+    _getlinkId: function(collection, i18Name) {
         return OpenLayers.i18n(i18Name, {id: collection.uuid});
     },
 
-    _removeButtonOnClick: function(button) {
-        var collectionId = this.getIdFromButtonContainerId(button, "removeButtonId");
+    _removeOnClick: function(containerId) {
```

Comment:
This should probably be called _removeLinkOnClick because we refer to the element as this.removeLink in other places.